src/components/ebay-date-textbox/index.marko

```diff
@@ -0,0 +1,49 @@
+// @ts-check
+
+<span.date-textbox
+    on-expander-expand("openPopover")
+    on-expander-collapse("closePopover")>
+    <if(input.range)>
```

Comment:
I think we only need the if statement to render one of the textboxes. They are so similar that we can probably render one by default and the second only when this is true.

Reply:
Ok, I changed some logic around to make this work